src/index.ts

```diff
@@ -38,9 +40,10 @@ const printHelp = (exitCode: number) => {
 
 {bold OPTIONS}
 
-  -V, --version            output version number
-  -h, --help               output usage information
-  -w, --write              restructure and edit folders and files
+  -V, --version               Output version number
+  -h, --help                  Output usage information
+  -w, --write                 Restructure and edit folders and files
+  -r, --rename-only-child     Flag to indicate if single files in folders are allowed
```

Comment:
https://github.com/benawad/destiny/pull/89#discussion_r386085311
```suggestion
-S, --avoid-single-file      Disallow directories with a single file and instead rename files
```